plugins/fcp_can_c/templates/can_signal_parser.h

```diff
@@ -0,0 +1,126 @@
+#ifndef CAN_SIGNAL_PARSER_H
+#define CAN_SIGNAL_PARSER_H
+
+#include <stdint.h>
+
+#include "can_frame.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+uint64_t can_decode_signal_as_uint64_t(const CanFrame *msg, uint64_t start,
```

Comment:
We have to add more comprehensive tests for these files.

We can leave that for another PR.